src/main/java/com/google/javascript/gents/TypeScriptGenerator.java

```diff
@@ -225,6 +225,11 @@ public CodeGenerator getCodeGenerator(Format outputFormat, CodeConsumer cc) {
                 .setOutputTypes(true)
                 .build();
 
+        // For whatever reason closure always prefixes the emit with an empty new line.
+        // Technically, this will not respect if the original .js intentionally had a new line
+        // to begin with, but I would argue that's a bad style.
+        if (tsCode.startsWith("\n")) tsCode = tsCode.substring(1);
```

Comment:
nit: Remove all leading `\n`s?

I feel if we care more about avoiding the bugs caused by the leading empty newline we should remove all leading empty newlines. If we care more about respecting the original .js we should remove all newlines added by Closure.

Reply:
Done, now we remove only the ones added by closure. Take another look.